{"name": "test_loginTestsOne", "status": "broken", "statusDetails": {"message": "AttributeError: 'Test_TestOne' object has no attribute 'customer1'", "trace": "self = <Tests.test_login.Test_TestOne object at 0x0000015155420950>, setup = <selenium.webdriver.edge.webdriver.WebDriver (session=\"e4f9e3e76b6ea951fa8cd32d5ebe88c2\")>\n\n    @pytest.mark.consy\n    @allure.severity(allure.severity_level.CRITICAL)\n    def test_loginTestsOne(self, setup):\n        self.driver = setup\n        self.driver.get(self.bankingURL)\n        self.driver.maximize_window()\n        self.login = LoginPage(self.driver)\n        self.login.clickCustomerLoginButton()\n>       self.login.enterCustomerName(self.customer1)\nE       AttributeError: 'Test_TestOne' object has no attribute 'customer1'\n\nTests\\test_login.py:20: AttributeError"}, "start": 1729618131023, "stop": 1729618143461, "uuid": "460ee07b-3a3f-4c75-97b8-d7c0d17e7ebc", "historyId": "16354c69e511ecc93a3c1ceee382e6a5", "testCaseId": "16354c69e511ecc93a3c1ceee382e6a5", "fullName": "Tests.test_login.Test_TestOne#test_loginTestsOne", "labels": [{"name": "severity", "value": "critical"}, {"name": "tag", "value": "consy"}, {"name": "parentSuite", "value": "Tests"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "Test_TestOne"}, {"name": "host", "value": "ZAJ-TUTJACONS"}, {"name": "thread", "value": "20472-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Tests.test_login"}]}